### PR TITLE
Authentication logs guide

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Adds the new authentication logs guide

### Motivation
We have a new blog post that goes in conjunction with this 😸 

### Preview
https://docs-staging.datadoghq.com/sarina/authenticating-logs-guide/security_monitoring/guide
https://docs-staging.datadoghq.com/sarina/authenticating-logs-guide/security_monitoring/guide/monitor-authentication-logs-for-security-threats/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_monitoring/guide/monitor-authentication-logs-for-security-threats.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+Log collection must be enabled to use Security and Compliance Monitoring. This guide recommends log collection enabled at the [application level][2].
+
+## Manage and format authentication logs
+
+Before you begin monitoring for security threats, follow the best practices below to ensure you have sufficient log data flowing into Datadog.
+
+### Log events from all login flows
+
+In order to gain visibility into all of your authentication activity, ensure log events for all login flows are at the application level. This eliminates gaps in your monitoring coverage. It also gives you more control over how you log authentication events and what data you are collecting.
```
Comment (cswatt) [inline content/en/security_monitoring/guide/monitor-authentication-logs-for-security-threats.md:26]:
```suggestion
To gain visibility into all of your authentication activity, ensure log events for all login flows are at the application level. This eliminates gaps in your monitoring coverage. It also gives you more control over how you log authentication events and what data you are collecting.
```

File: content/en/security_monitoring/guide/monitor-authentication-logs-for-security-threats.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+### Ensure your logs contain useful data
+
+By logging all authentication events at the application level, you can ensure that your logs contain the most useful data for [monitoring and detecting security threats](#monitor-and-detect-security-threats).
+
+{{< code-block lang="bash" >}}
+2020-01-01 12:00:01 google oauth login success by John Doe from 1.2.3.4
+{{< /code-block >}}
+
+Logs that contain the "who" (John Doe), "what" (login success), and when (2020-01-01 12:00:01) of an event provide the best detail for you to perform complex analysis in Datadog.
```
Comment (cswatt) [inline content/en/security_monitoring/guide/monitor-authentication-logs-for-security-threats.md:36]:
```suggestion
Logs that contain the "who" (John Doe), "what" (login success), and when (2020-01-01 12:00:01) of an event provide the best level of detail for you to perform complex analysis in Datadog.
```

File: content/en/security_monitoring/guide/monitor-authentication-logs-for-security-threats.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+A key-value format also makes it easy to add custom attributes to logs. For example, adding a [reCAPTCHA v3][9] score to identify possible bot activity. Use quotes to wrap any attribute values that may contain spaces. This ensures that you capture the full value in a way that is easily parsable.
+
+## Monitor and detect security threats
+
+To properly monitor and detect security threats, there are key patterns you should take note of. For example, if you see a significant number of failed login attempts from a single user within a short period of time, it could indicate a [**brute force attack**][10]. If those failed login attempts are followed by a successful one, it could be a successful account takeover that you should investigate immediately.
+
+Another common authentication attack technique is [**credential stuffing**][11]. Credential stuffing is when an attacker mixes and matches breached login credentials to try to match a real user account. In order to detect this type of attack, look for logins using multiple `usr.id` values all coming from the same `network.client.ip`.
```
Comment (cswatt) [inline content/en/security_monitoring/guide/monitor-authentication-logs-for-security-threats.md:82]:
```suggestion
Another common authentication attack technique is [**credential stuffing**][11]. Credential stuffing is when an attacker mixes and matches breached login credentials to try to match a real user account. To detect this type of attack, look for logins using multiple `usr.id` values all coming from the same `network.client.ip`.
```

File: content/en/security_monitoring/guide/monitor-authentication-logs-for-security-threats.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+{{< img src="security_monitoring/guide/monitor-authentication-logs-for-security-threats/credential-stuffing-attack-signal.png" alt="A credential stuffing attack signal in the Security Signals Explorer">}}
+
+Datadog offers pre-configured [Detection Rules][12] which scan your ingested logs in real time for common attacker techniques like these two. If any log triggers one of these rules, it automatically generates a [Security Signal][13] which includes key data about the event, such as the type of attack detected and suggestions on how to respond and remedy the situation. You can view, filter, and sort all of your Security Signals in the explorer to triage them and see where to best focus your efforts.
```
Comment (cswatt) [inline content/en/security_monitoring/guide/monitor-authentication-logs-for-security-threats.md:86]:
```suggestion
Datadog offers pre-configured [Detection Rules][12] that scan your ingested logs in real time for common attacker techniques like the two mentioned above. If any log triggers one of these rules, Datadog automatically generates a [Security Signal][13]. This signal includes key data about the event, such as the type of attack detected and suggestions on how to respond and remedy the situation. You can view, filter, and sort all of your Security Signals in the explorer to triage them and see where to best focus your efforts.
```

File: content/en/security_monitoring/guide/monitor-authentication-logs-for-security-threats.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+For signals triggered from the `Credential Stuffing Attack` Detection Rule, there is an [out-of-the-box runbook][14] available to help with response and remediation. This runbook guides you through investigating a potential credential stuffing attack, and includes graphs of related logs. To use this runbook, save a copy and set the the time frame, document your investigation in markdown, and share it with teammates [for commenting][15].
```
Comment (cswatt) [inline content/en/security_monitoring/guide/monitor-authentication-logs-for-security-threats.md:88]:
```suggestion
For signals triggered from the `Credential Stuffing Attack` Detection Rule, there is an [out-of-the-box runbook][14] available to help with response and remediation. This runbook guides you through investigating a potential credential stuffing attack and includes graphs of related logs. To use this runbook, save a copy and set the the time frame, document your investigation in markdown, and share it with teammates [for commenting][15].
```

File: content/en/security_monitoring/guide/monitor-authentication-logs-for-security-threats.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+### Use dashboards to investigate
+
+Datadog provides out-of-the-box dashboards, such as the [IP investigation dashboard][16] and [User investigation dashboard][17]. These correlate key data from your authentication logs with relevant data from the rest of your environment to assist in your investigations.
```
Comment (cswatt) [inline content/en/security_monitoring/guide/monitor-authentication-logs-for-security-threats.md:92]:
```suggestion
Datadog provides out-of-the-box dashboards, such as the [IP investigation dashboard][16] and [user investigation dashboard][17]. These correlate key data from your authentication logs with relevant data from the rest of your environment to assist in your investigations.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
just grammar/style things—looks really good!

**Comment by sarina-dd** [comment]
@cswatt committed all changes, thanks for the review!

**Review by cswatt — APPROVED** [review]

**Pull request merged**